.gitignore

```diff
@@ -40,4 +40,8 @@ example/extension/nethttp/demo
 example/log/test
 test/build/cmd/cmd
 ssa.html
-otel_pkg
\ No newline at end of file
+otel_pkg
+ssa.html
+otel_importer.go
+test/fiberv2/v2.43.0/fiber_http
+test/redis/v9.0.5/test_redis_ring
```

Comment:
Do we need to add those test binary here?

Reply:
It's okay, gitignore is somewhat flexible, any unnecessary files generated during the development can be added to it.